### DISCUSSION TITLE
Issue with badges in desktop version

### DISCUSSION THREAD

**Discussion by KQ29** [community/community#142720] [Other Feature Feedback, Questions, & Ideas]
Hello Everyone. 

I faced with the problem related to the badges in my profile. In mobile application, badges are visible, but in the website it is not. Can somebody explain how to fix it?

**Comment by VishalKotekar** [comment]
Hey there,

It sounds like a frustrating issue with your profile badges not showing up on the desktop version. Here are a few things you can try:

Clear your browser's cache and cookies.
Make sure your browser is up to date.
Temporarily disable any browser extensions that might be causing conflicts.
If these steps don't help, it might be worth reaching out to GitHub support for more detailed assistance.

Hope this helps!
